src/openMarket/infrastructure/order/OrderFactoryImpl.js

```diff
@@ -7,11 +7,11 @@ export default class OrderFactoryImpl extends OrderFactory {
     this._identity = identity;
   }
 
-  createWith({lines}) {
-    return new Order(
-      {
-        idGenerator: this._identity,
-        lines: lines
+  createWith({id = this._identity.generate(), lines, date} = {}) {
```

Comment:
id is given and if not the default value is the identity generate? Just to understand

Reply:
Yes, is like the previous case in domain Order constructor :+1: